.env

```diff
@@ -71,3 +71,6 @@ REACT_APP_PATH_REGEX_ENS="/ipfs"
 
 # Enables mock mode (default = true)
 REACT_APP_MOCK=true
+
+# Gp Api
+REACT_APP_DEFAULT_GP_API=LEGACY
```

Comment:
Can we give it a more meaningful name? 

This is the price querying strategy, so sth like:
REACT_APP_DEFAULT_PRICE_STRATEGY

Values: LEGACY, COWSWAP sounds good to me. Although now im thinkining if `DEFAULT_PRICE_STRATEGY = 'best-price-apis` and `cowswap-api` are better names for our price strategies

Reply:
i actually removed this, i don't think it's necessary as an environment variable and can just be used as a constant

Reply:
Fine with me too